### PR TITLE
Change the description of shell profile and add parameter -i in quick-start-with-tidb.md

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)
- Change the description of Shell profile
- Add parameter -i to specify the key file used to connect to the target machine
- Modify Mac to macOS
- Fix format

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v6.1 (TiDB 6.1 versions)
- [x] v6.0 (TiDB 6.0 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/9015
    https://github.com/pingcap/docs-cn/pull/9019
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: quick-start-with-tidb.md [hunk 1]
```diff
@@ -38,16 +38,30 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
...
+    Detected shell: zsh
+    Shell profile:  /Users/user/.zshrc
+    /Users/user/.zshrc has been modified to add tiup to PATH
+    open a new terminal or source /Users/user/.zshrc to use it
+    Installed path: /Users/user/.tiup/bin/tiup
+    ===============================================
+    Have a try:     tiup playground
+    ===============================================
+    ```
+
```
Comment (ran-huang) [inline quick-start-with-tidb.md:54]: We should tell the user to copy or at least remember the shell profile path here.

File: quick-start-with-tidb.md [hunk 2]
```diff
@@ -38,16 +38,30 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
...
     > **Note:**
     >
-    > After the installation, TiUP displays the absolute path of the corresponding `profile` file. You need to modify the following `source` command according to the path.
+    > After the installation, TiUP displays the absolute path of the corresponding Shell profile file. You need to modify `${your_shell_profile}` in the following `source` command according to the path.
 
     {{< copyable "shell-regular" >}}
 
     ```shell
-    source .bash_profile
+    source ${your_shell_profile}
```
Comment (ran-huang) [inline quick-start-with-tidb.md:64]: Can we add a sentence like:
```
In this case, `${your_shell_profile}` is `/Users/user/.zshrc` from the output of Step 1.
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: quick-start-with-tidb.md [hunk 3]
```diff
@@ -38,16 +38,32 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
...
+    Shell profile:  /Users/user/.zshrc
+    /Users/user/.zshrc has been modified to add tiup to PATH
+    open a new terminal or source /Users/user/.zshrc to use it
+    Installed path: /Users/user/.tiup/bin/tiup
+    ===============================================
+    Have a try:     tiup playground
+    ===============================================
+    ```
+
+    Please pay attention to the Shell profile path above. After installation, this file has been modified to add tiup to PATH. You can open a new terminal or follow the next step to declare the global environment variables of the TiUP.
```
Comment (ran-huang) [inline quick-start-with-tidb.md:55]: 1. Let's not overdo the politeness. ([ref](https://developers.google.com/style/tone))
2. "You can open a new terminal or follow the next step" might confuse the user.

```suggestion
    Note the Shell profile path in the output above. You need to use the path in the next step.
```

File: quick-start-with-tidb.md [hunk 4]
```diff
@@ -38,16 +38,32 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
     curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
     ```
 
+    Verify that TiUP has installed successfully by checking the output in the terminal or shell window for the following message:
```
Comment (shichun-0415) [inline quick-start-with-tidb.md:41]:
```suggestion
    If the following message is displayed, you have installed TiUP successfully:
```

File: quick-start-with-tidb.md [hunk 5]
```diff
@@ -141,16 +159,32 @@ As a distributed system, a basic TiDB test cluster usually consists of 2 TiDB in
     curl --proto '=https' --tlsv1.2 -sSf https://tiup-mirrors.pingcap.com/install.sh | sh
     ```
 
+    Verify that TiUP has installed successfully by checking the output in the terminal of shell window for the following message:
```
Comment (shichun-0415) [inline quick-start-with-tidb.md:162]:
```suggestion
    If the following message is displayed, you have installed TiUP successfully:
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**